pages/index.js

```diff
@@ -47,7 +46,7 @@ export default function Home(props) {
         <link rel='icon' href='/favicon.ico' />
       </Head>
       <DisplayBasicInfo data={data.main} />
-      <DisplayEventInfo result={eventData} />
+      <DisplayEventInfo result={props} />
```

Comment:
passing `props.result` would be a good idea because that's only thing what `DisplayEventInfo` matters.

Reply:
I also need `props.error` for DisplayEventInfo to show other content.🧐

Reply:
I think it's `props.result.error` 😃